software/script/chameleon_cli_unit.py

```diff
@@ -851,6 +851,41 @@ def on_exec(self, args: argparse.Namespace):
         return
 
 
+@hf_mf.command('fuzz')
+class HFMFFuzz(ReaderRequiredUnit):
+
+    key_re = re.compile(r"^(?P<idx>([0-9]|1[0-5])):(?P<ab>(A|B)):(?P<key>([0-9A-Fa-f]{12}))$")
+
+    def args_parse_key(self, arg):
+        m = cls.key_re.match(arg)
```

Comment:
Sorry, I had a typo in my code, this is not a class method!
```suggestion
        m = self.key_re.match(arg)
```